components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -80,9 +82,21 @@ export const PrebuildDetailPage: FC = () => {
             }
         });
         logEmitter.on("logs-error", (err: ApplicationError) => {
+            const phase = prebuild?.status?.phase?.name;
+            // An overcomplicated check for getting a "failed" prebuild state, for which we do not store logs.
+            if (
+                phase &&
+                ([PrebuildPhase_Phase.FAILED, PrebuildPhase_Phase.TIMEOUT].includes(phase) ||
+                    (phase === PrebuildPhase_Phase.AVAILABLE && prebuild.status?.message))
+            ) {
+                setLogNotFound(true);
```

Comment:
We could check error code if it's ApplicationError + ErrorCodes.NotFound and add this guidance

Reply:
Fixed in https://github.com/gitpod-io/gitpod/pull/19460/commits/11b33f6ffda38a6b18b29e35ca248e956bcd121a